dbxio/delta/table_commands.py

```diff
@@ -37,24 +37,39 @@ def exists_table(table: Union[str, Table], client: 'DbxIOClient') -> bool:
         return False
 
 
-def create_table(table: Union[str, Table], client: 'DbxIOClient') -> _FutureBaseResult:
-    """
-    Creates a table in the catalog.
-    If a table already exists, it does nothing.
-    Query pattern:
-        CREATE TABLE IF NOT EXISTS <table_identifier> (col1 type1, col2 type2, ...)
-        [USING <table_format> LOCATION <location>]
-        [PARTITIONED BY (col1, col2, ...)]
-    """
+def _create_table_query(table: Union[str, Table], replace: bool, if_not_exists: bool, include_schema: bool) -> str:
     dbxio_table = Table.from_obj(table)
 
-    schema_sql = ','.join([f'`{col_name}` {col_type}' for col_name, col_type in dbxio_table.schema.as_dict().items()])
-    query = f'CREATE TABLE IF NOT EXISTS {dbxio_table.safe_table_identifier} ({schema_sql})'
+    if include_schema and dbxio_table.schema:
+        schema_sql = f'({dbxio_table.schema.as_sql()})'
+    else:
+        schema_sql = ''
+    if replace:
+        query = 'CREATE OR REPLACE TABLE'
+    else:
+        query = 'CREATE TABLE'
+        if if_not_exists:
+            query += ' IF NOT EXISTS'
+    query += f' {dbxio_table.safe_table_identifier} {schema_sql}'
```

Comment:
```suggestion
    schema_sql = dbxio_table.schema.as_sql() if include_schema and dbxio_table.schema else ''
    query = 'CREATE OR REPLACE TABLE' if replace else f'CREATE TABLE {"IF NOT EXISTS" if if_not_exists else ""}'
    query += f' {dbxio_table.safe_table_identifier} ({schema_sql})'
```

Reply:
Let's force even schema existence here bc we can face some problems on write without explicit schema declaration

Reply:
We can't force schema for `Create Table As Select (CTAS) statement`